e2e/test/e2e.test.ts

```diff
@@ -0,0 +1,58 @@
+import { ExplorerProvider, NodeProvider } from '@alephium/web3'
+import { PrivateKeyWallet } from '@alephium/web3-wallet'
+import  configuration  from './config'
+
+describe('e2e', () => {
+
+  const nodeProvider = new NodeProvider(configuration.nodeUrl)
+  const explorerProvider = new ExplorerProvider(configuration.explorerUrl)
+
+  const oneAlph = 10n ** 18n
+
+  const wallet = PrivateKeyWallet.FromMnemonic({
```

Comment:
For all the places where we use `PrivateKeyWallet`, maybe we can use `getSigner` instead.